use-cases/revenue-management.md

```diff
@@ -8,23 +8,30 @@ Performed once when setting up the connection, because the RMS needs to obtain h
 
 * Open: Items which have not been closed on a specific bill.
```

Comment:
BTW I think this should rather be defined directly in docs for the accounting state enum, because it's valuable for all types of integrations that use it. We don't need to duplicate it here, we can link it here of course.

Reply:
good point, will update